src/rf/requirements/workers.txt

```diff
@@ -1,2 +1,3 @@
 GDAL==1.10.0
 numpy==1.10.1
+exifread==2.1.2
```

Comment:
To run migrations I typically do `./scripts/manage.sh migrate` which runs in the context on the app VM. Since this dependency only exists on the worker VM I get the following error:

```
ImportError: No module named exifread
```

We should consider consolidating the workers and app requirements.